docs/windows.md

```diff
@@ -426,6 +368,130 @@ antrea-agent-windows-6hvkw                             1/1     Running     0
 kube-proxy-windows-2d45w                               1/1     Running     0          102s
 ```
 
+### Add Windows kube-proxy DaemonSet (only for Kubernetes versions prior to 1.26)
+
+Starting from Kubernetes 1.26, Antrea will no longer support Windows kube-proxy
+because kube-proxy userspace mode is deprecated and kernelspace mode does not
+work with Antrea. You should skip this step on the kubernetes cluster starting
```

Comment:
Should we say: "For a cluster with Kubernetes version 1.26 or later, please refer to [Deploying Antrea on Windows Worker Node]"?

Reply:
Updated.